mcf-browser/src/FileEntry.tsx

```diff
@@ -34,41 +34,7 @@ interface FileEntryPropType {
    * Sets whether the dropdown on the home page for additonal file entries
    * should be displayed.
    */
-  toggle: Function;
-}
-
```

Comment:
I forget what @beets wanted to do for this.  Was she okay deleting?

Reply:
Did she mention something about it in a previous PR?

Reply:
Thought you asked a question in the meeting and she said something. Or was it about something else?  In any case, please wait for Jennifer to take a look before merging...

Reply:
Sounds good